tools/tool-base/src/main/java/io/spine/tools/java/code/UuidMethodFactory.java

```diff
@@ -31,7 +31,6 @@
 import com.squareup.javapoet.MethodSpec;
 import com.squareup.javapoet.ParameterSpec;
 import com.squareup.javapoet.TypeName;
-import io.spine.code.gen.java.JavaPoetName;
```

Comment:
That's interesting. I mean it somehow lasted through all these years, and almost escaped inevitable death.

So close :)

Reply:
Checkstyle helped.